lib/module/raspberry-pi.yaml

```diff
@@ -0,0 +1,47 @@
+datasheet: https://pinout.xyz/
+description: Raspberry Pi GPIO Connector
+designator: J
+footprint: agg:DIL-254P-40
+name: RASPBERRY-PI
+ordercodes: []
+pins:
+- - - [VCC-3V3, 1, pwrout]
+    - [VCC-3V3, 17, pwrout]
+    - [VCC-5V, 2, pwrin]
+    - [VCC-5V, 4, pwrin]
+  - - [GND, 6, pwrin]
+    - [GND, 9, pwrin]
+    - [GND, 14, pwrin]
+    - [GND, 20, pwrin]
+    - [GND, 25, pwrin]
+    - [GND, 30, pwrin]
+    - [GND, 34, pwrin]
+    - [GND, 39, pwrin]
+  - - [GPIO5, 29, bidi]
+    - [GPIO6, 31, bidi]
+    - [GPIO12, 32, bidi]
```

Comment:
does it make more sense to call GPIO12 PWM0, or even both this and GPIO18 as PWM0? pinouts.xyz lists both, and PWM1 is on GPIO13 which is physically near GPIO12 (nicer if you're using both PWMs I guess)